docs/docs/get-started/installing-nebari.md

```diff
@@ -45,6 +44,13 @@ There are currently two ways to install Nebari:
    mamba install nebari
    ```
 
+:::note
+The version of the Nebari your `nebari-config.yaml` must match your Nebari deployer package version.  See [Upgrading Nebari][nebari-upgrade] for techniques for upgrading your Nebari CLI or safely updating your older config file to match your Nebari package version.
```

Comment:
```suggestion
The version of Nebari in your `nebari-config.yaml` must match your current installed Nebari package version; otherwise, a warning will be raised when attempting to deploy. See [Upgrading Nebari][nebari-upgrade] for techniques for upgrading your Nebari CLI or safely updating your older config file to match your Nebari package version.
```

Reply:
Done